nerdlandbot/helpers/constants.py

```diff
@@ -18,7 +18,12 @@
 POLL_MAX_TIMEOUT = 60 #minutes
 POLL_DEFAULT_TIMEOUT = 5 #minutes
 
-EIGHT_BALL_URL = 'https://evening-castle-06130.herokuapp.com/eightball'
+
+# Reminder
+MAX_REMINDER_TIME = 1440 #minutes, 1 day
+MIN_REMINDER_TIME = 5 #minutes
+REMINDER_CONVERT_TO_MINUTES = 60
```

Comment:
This feels like a rather useless constant, since I assume that a minute will always have 60 seconds?